test/libsolidity/syntaxTests/constructor/constructor_state_mutability_new.sol

```diff
@@ -1,9 +1,13 @@
 contract test1 {
-  constructor() view {}
```

Comment:
Previously it was: view, pure default. Now it is constant, view, pure. Did you change that on purpose?

Reply:
This happened by accident while rebasing. It seems that you've commented on the old commit, because I fixed it already before.